mm2src/mm2_main/src/lp_network.rs

```diff
@@ -216,6 +216,76 @@ async fn process_p2p_message(
                 }
             }
         },
+        Some(lp_healthcheck::PEER_HEALTHCHECK_PREFIX) => {
+            macro_rules! try_or_return {
```

Comment:
Can code be put into a separate function like 'process_health_check messages?

Reply:
[Done](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2194/commits/13fe925e3c62c95b980de8ec279119a8593e0f70).